docs/_includes/ds6/symbols.html

```diff
@@ -225,4 +222,16 @@
     <symbol id="icon-window" viewBox="0 0 28 25">
         <path d="M.497 5.701h15.644c.274 0 .497.222.497.497v.718a.497.497 0 0 1-.497.497H.497A.497.497 0 0 1 0 6.916v-.718c0-.274.222-.497.497-.497zm1.215 0V23.94H0V5.701h1.712zm18.515 7.785v9.939a.552.552 0 0 1-.552.552h-.699a.552.552 0 0 1-.552-.552v-9.939c0-.305.247-.552.552-.552h.699c.305 0 .552.247.552.552zM0 22.302h20.209v1.712H0v-1.712zM25.767.161h1.914v16.583h-1.914V.161zm1.914-.147v1.914H11.098V.014h16.583zM26.322 0l1.353 1.353L13.398 15.63l-1.353-1.353L26.322 0z"/>
     </symbol>
+    <symbol id="icon-confirmation" viewBox="0 0 14 14">
+		<path fill="#5ba71b" xlink:href="#a" fill-rule="evenodd"
```

Comment:
I don't believe we need `xlink:href="#a"`.

Reply:
good call 👍 . Removing `fill-rule` as well. Learning curve I guess 😄

Reply:
Yeah, a lot of these are extraneous from the perspective of the SVG output itself. I'm grateful for SVGOMG, but it still misses some of these, too.

Reply:
Sorry, should have clarified: `fill-rule` is needed. `xlink:href="#a"` is not